Examples/Algorithms/Geant4/include/ActsExamples/Geant4/Geant4Options.hpp

```diff
@@ -0,0 +1,43 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2020 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include "ACTFW/Utilities/Options.hpp"
+#include "ACTFW/Utilities/OptionsFwd.hpp"
+#include "Acts/Utilities/Logger.hpp"
+#include "Acts/Utilities/Units.hpp"
+
+#include <iostream>
```

Comment:
This include is not needed in the header. I am not sure if it is needed at all, but if it is and include in the `.cpp` suffices.

Reply:
It's not needed at all, it was a temporary hack to print something out.